src/strings-in-block/class-base.php

```diff
@@ -87,6 +87,9 @@ public static function get_string_type( $string ) {
 		if ( strpos( $string, '<' ) !== false ) {
 			$type = 'VISUAL';
 		}
+		if ( strpos( $string, '://' ) !== false ) {
```

Comment:
@GeorgeBotsev this does not guarantee this will handle only links. Should this mark strings like "Page addresses always have :// after the protocol name" as LINK too?

I suggest:
```
if (filter_var($string, FILTER_VALIDATE_URL) !== false)
```

Reply:
George, please fix this and do another `git commit -a -m '[fixup] better url validation'`